### DISCUSSION TITLE
[bug] dropdownMenu 属性类型不匹配

### DISCUSSION THREAD

**Discussion by chenzylab** [hngngn/shadcn-solid#92] [General]
[代码位置](https://github.com/hngngn/shadcn-solid/blob/217370c3b87dd03d19a1cb757b6110278b47e30f/packages/unocss/ui/dropdown-menu.tsx#L91)
![image](https://github.com/hngngn/shadcn-solid/assets/16700482/4711fbc3-d0fb-4db5-bc80-1808b235e942)

错误描述：Type '{ id?: string | undefined; ref?: HTMLElement | ((el: HTMLElement) => void) | undefined; as: "div"; class: string; }' is not assignable to type 'IntrinsicAttributes & Omit<HTMLAttributes<HTMLDivElement>, "as" | keyof MenuItemLabelCommonProps<ElementOf<T>>> & MenuItemLabelOptions & Partial<...> & PolymorphicAttributes<...>'.
  Type '{ id?: string | undefined; ref?: HTMLElement | ((el: HTMLElement) => void) | undefined; as: "div"; class: string; }' is not assignable to type 'Partial<MenuItemLabelCommonProps<HTMLDivElement>>'.
    Types of property 'ref' are incompatible.
      Type 'HTMLElement | ((el: HTMLElement) => void) | undefined' is not assignable to type 'HTMLDivElement | ((el: HTMLDivElement) => void) | undefined'.
        Type 'HTMLElement' is not assignable to type 'HTMLDivElement | ((el: HTMLDivElement) => void) | undefined'.
          Property 'align' is missing in type 'HTMLElement' but required in type 'HTMLDivElement'.ts(2322)
lib.dom.d.ts(10066, 5): 'align' is declared here.

**Comment by hngngn** [comment]
create a new issue please, thanks!
